src/core/main.cpp

```diff
@@ -427,6 +428,22 @@ int main( int argc, char * * argv )
 
 			return EXIT_SUCCESS;
 		}
+		else if( arg == "compress" || arg == "--compress" )
```

Comment:
Can you update the code formatting to the [Coding Conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions) please?

Just some whitespace changes is all. 😄

Reply:
But it's adopting to the surrounding code so I think it's good like this.